### PR TITLE
fixed typos/rewording

### PR DESCRIPTION
**Pull request by DoctorCr4nk**
I fixed minor typos and reworded some stuff.
also I merged the systemd commands into one

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,18 @@
 # bat-status
 Check battery capacity and send desktop notification via `notify-send`
-if it gets low.
+if it reaches certain charge levels.
```
Comment (phkam) [inline README.md]: how about:` if it falls below a certain charge level` ?

File: README.md [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
 ## Install and run
 
 Either run the script yourself, put it in your autostart or use the systemd
 service and timer which are part of this repository.
 If you use the timer, install the shell script to `/opt/bat-status/bat-status.sh`
 and put the `.service` and `.timer` files in `/etc/systemd/system/`.
-Then you need to `systemctl daemon reload` and activate the timer with
-`systemctl start bat-status.timer`. Also enable the unit to load on system start
-`systemctl enable bat-status-timer`.
+To load the new systemd files you have to reload the daemon with `systemctl daemon-reload`. You can start the service and enable it on system startup with `systemctl enable --now bat-status.timer`.
```
Comment (phkam) [inline README.md]: Please keep the line length reasonable

**Review by phkam — CHANGES_REQUESTED** [review]
Thanks for spotting all these typing errors. I figured using "charge" would cause less confusion, since the file is called this way. But from a technical point of view charge or level sound much better.
Please consider the comments I made. Otherwise I’m happy to merge it.

File: README.md (hunk 1 above)
Comment (junglert) [reply README.md]: I think falls is not the correct word to use here

I would got with reaches aswell.

Comment (phkam) [reply README.md]: Perhaps `if it drops below…` is acceptable?

Comment (junglert) [reply README.md]: `if it drops below a certain charge level`

**Review by phkam — APPROVED** [review]

**Pull request merged**